dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Machines/ReactivePowerControlLoops/ReactivePowerControlLoop2.mo

```diff
@@ -0,0 +1,122 @@
+within Dynawo.Electrical.Controls.Machines.ReactivePowerControlLoops;
+
+/*
+* Copyright (c) 2022, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+model ReactivePowerControlLoop2 "Simplified Reactive Power Control Loop model for renewables"
+  /* The reactive control loop gets a level K from the secondary voltage control and transforms it to a voltage reference for the generator voltage regulator */
+  import Modelica;
+  import Dynawo.Types;
+
+  parameter Types.PerUnit QrPu "Participation factor of the generator to the secondary voltage control in pu (base QNom)";
+  parameter Types.PerUnit DeltaURefMaxPu "Maximum of deltaUref on one activation period (Uref(t+1) - Uref(t)) (base UNom)";
```

Comment:
Maximum of deltaU**R**ef on one activation period (U**R**ef(t+1) - U**R**ef(t)) **in pu** (base UNom)